app/templates/gentelella/users/events/sessions/base_session_table.html

```diff
@@ -233,7 +233,7 @@ <h4 class="modal-title" id="mailModalLabel">Customize
             });
 
         var table = $('.with-datatable').DataTable({
-            "dom": '<"row"<"toolbar col-md-7"<"pull-left"l>><"col-md-3 pull-right"f>>tip',
+            "dom": '<"row"<"toolbar col-md-7"<"pull-left"l>><"col-md-3 pull-right"f>>tip', "pageLength": 50,
```

Comment:
@sudheesh001 could you explain a bit ? That's as per datatables documentation ... AFAIK, that's a right way to increase page length ...